client-web/src/functional-e2e/authentication/experiment.spec.ts

```
@@ -0,0 +1,59 @@
+test.afterAll(async () => {
+  await TestScenarioFactory.cleanUpBaseScenario(baseScenario);
+});
+
+test.beforeEach(async ({ context }) => {
+  await context.clearCookies();
+  await deleteMailSlurperMails();
+});
+
+test.describe.configure({ mode: 'serial' });
+
+test.only('Experiment - client test with scenario setup', async ({ page }) => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)